extra/1-radio-stations.js

```diff
@@ -15,6 +15,24 @@
 
 // `getAllFrequencies` goes here
 
+function getAllFrequencies() {
+  // return [87,88,89,90,91,92,93,94,95,96,97,98,99,100,101,102,103,104,105,106,107,108];
+
+  // let array = [];  
+  // for (let i = 87; i <= 108; i++) {
+  //   array.push[i];
+  // }
+  // return array;
+
+  return Array.from({length: 108 - 87 + 1}, (v, i) => i + 87);
```

Comment:
This is smart, but you're relying on the reader of the code to know the intricacies of the `Array.from` method. It's quite rare that I see that in production, and reading this it's not immediately apparent what's going on without referring to the spec.

Under the hood here, you're basically tricking JS into thinking that the object you're passing is iterable by giving it a length property, and it's creating an array by iterating over your fake iterable and pushing the number into an array. I think it's good to learn these new syntaxes and to experiment like this, but be mindful that in some cases (and particularly when there's no efficiency gain) it's possible to overengineer a solution and make it less readable and maintainable.

Reply:
yes, it's excessive 🤣

but i guess i wanted to play with Array.from() because i see so many people use it on CodeWars in weird and wonderful ways... so i had a go here

I can't claim to understand the full depths of Array.from() but this was a chance to try to build an array fast 😁